src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineDiagnostics
+{
+    internal static class InlineDiagnosticsOptions
+    {
+        public static readonly PerLanguageOption2<bool> EnableInlineDiagnostics =
+            new(nameof(InlineDiagnosticsOptions),
+                nameof(EnableInlineDiagnostics),
+                defaultValue: false,
+                storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.InlineDiagnostics"));
+
+        public static readonly PerLanguageOption2<InlineDiagnosticsLocations> LocationOption =
```

Comment:
nit.  no need to end with `Option` in the name.

Reply:
removed